scanners/httpobs_scanner.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import time
+
+
+class HTTPObservatoryScanner():
+    ## TODO: Re-write this with the httpobs python module
+    def __init__(self, sleep_interval=1):
+        self.session = requests.Session()
+        self.sleep_interval = sleep_interval
+        self.api_url = 'https://http-observatory.security.mozilla.org/api/v1'
```

Comment:
Should this be an environment variable, rather than being hard coded into the class?

Reply:
I see HTTPOBS_URL in the below serverless.yml, is that what we should be using?

Reply:
Yes, I wasn't using that env variable in serverless config yet, but will change it to the above as it makes more sense.